asset_dashboard/models.py

```diff
@@ -221,33 +237,26 @@ class Meta:
         verbose_name_plural = 'Project Scores'
 
 
-class PhaseFinances(models.Model):
-
-    FUNDING_CHOICES = [
-        ('unfunded', 'Unfunded'),
-        ('partially funded', 'Partially Funded'),
-        ('future unfunded', 'Future Unfunded'),
-        ('funded', 'Funded')
+class FundingStream(models.Model):
+    SOURCE_TYPE_CHOICES = [
+        ('capital_improvement_fund', 'Capital Improvement Fund'),
+        ('bonds', 'General Obligation Bonds'),
+        ('construction_development', 'Construction & Development'),
+        ('grants_fees_other', 'Grants, Fees, & Other'),
+        ('rollover', 'Rollover')
     ]
 
-    phase = models.OneToOneField('Phase', on_delete=models.CASCADE)
     budget = MoneyField(default_currency='USD',
                         default=0.00,
                         max_digits=11)
+    obligated_year = models.IntegerField(null=True, blank=True)
+    # TODO: better model field type? do we need months/days?
+    obligated_completion_date = models.IntegerField(null=True, blank=True)
```

Comment:
Do we need months/days or does an IntegerField work here (so we only save year)?